config/gulp/javascript.js

```diff
@@ -27,6 +27,42 @@ gulp.task('eslint', function (done) {
 
 });
 
+gulp.task('copy-docs-javascript', function (done) {
+
+  dutil.logMessage(task, 'Copying JS from js/');
+
+  var stream = gulp.src('./js/**/*')
+    .pipe(gulp.dest('assets/js/'));
+
+  return stream;
+
+});
+
+gulp.task('copy-jquery-javascript', function (done) {
```

Comment:
Oh, duh, sorry. We're saying the same thing. No, you're right, it doesn't need to be a task because it's already in `package.json` and so Browserify will pull it in anyway.